client/app/_components/layout/header/Header.tsx

```diff
@@ -3,18 +3,38 @@ import Link from 'next/link'
 import { Button } from '@/_components/ui'
 import { SettingDropdown } from '..'
 
-const Header = () => {
+type HeaderProps = {
+  hasNavigation?: boolean
+}
+
+const Header = ({ hasNavigation = true }: HeaderProps) => {
```

Comment:
Now three type of Header we can use.
1. No header for login and register pages
2. Has header but can not navigate for House setting create page
3. Complete header for all protect pages